src/index.test.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {
+    AxeResults,
+    Result,
+    RunOptions,
+    TestEngine,
+    TestEnvironment,
+    TestRunner,
+} from 'axe-core';
+import * as fs from 'fs';
+import { axeToSarif } from './index';
+import { SarifLog } from './sarif/sarifLog';
+import { Run } from './sarif/sarifv2';
+
+describe('axe-sarf-converter integration test', () => {
+    let axeResultStub: AxeResults;
```

Comment:
I think this makes more sense as a local const in the single test dependent on it until you actually have a second test that warrants extracting it.

If you feel strongly that it should be defined outside that test, it should be a const defined in place, rather than a mutable variable; you don't need to use `beforeEach` here.

Reply:
I used `beforeEach` to avoid the tslint warning. But I can obviously make it a constant outside the test since its constant.